h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -332,7 +336,6 @@ private void notifyModelListeners() {
   public Job<M> trainModelOnH2ONode() {
     if (error_count() > 0)
       throw H2OModelBuilderIllegalArgumentException.makeFromBuilder(this);
-    this._input_parms = (P) this._parms.clone();
```

Comment:
@koniecsveta I think this one was a mistake as we're cloning AFTER validation, during which many parms are set or modified internally.